test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -2669,36 +2661,6 @@ protected ResultSet executeNet(ProtocolVersion protocolVersion, org.apache.cassa
         return sessionNet(protocolVersion).execute(stmt.toCQL(), stmt.bindsEncoded());
     }
 
-    protected Mutation nonTransactionMutation(RandomSource rs, TableMetadata metadata)
-    {
-        return Generators.toGen(new ASTGenerators.MutationGenBuilder(metadata).withoutTransaction().build()).next(rs);
-    }
-
-    protected Select select(Mutation mutation)
-    {
-        // select * from table where <primaryKeys>
-        return new Select(Collections.emptyList(),
-                          Optional.of(new TableReference(Optional.of(mutation.table.keyspace), mutation.table.name)),
-                          where(mutation.primaryKeys()),
```

Comment:
supporting `mutation.primaryKeys()` is actually annoying as a mutation can impact multiple rows... This is also trying to hack together a bad model, so with the next patch adding a model it makes more sense to drop this logic in favor of the pending model.